src/mero_wrk_tcp_binary.erl

```diff
@@ -137,8 +137,38 @@ transaction(Client, async_mget, [Keys]) ->
             {Client, ok}
     end;
 
-transaction(Client, async_mget_response, [Keys, Timeout]) ->
-    case async_mget_response(Client, Keys, Timeout) of
+transaction(Client, async_increment, [Keys]) ->
+    case async_increment(Client, Keys) of
```

Comment:
I suggest to use the same function for them all and do case transaction(Client, Op, Args) ->
 +    case apply(?MODULE, Op, [Client | Args]) of